config/datastores/posixDatastore.yaml

```diff
@@ -1,6 +1,6 @@
 datastore:
   cls: lsst.daf.butler.datastores.posixDatastore.PosixDatastore
-  root: default_butler_repository
+  root: <root>/datastore
```

Comment:
Why `<root>`, rather than something more familiar like `{root}`, `$root`, or `%(root)s`?

Reply:
I wanted something that looked a bit like a HTML tag that stood out as obviously not being a file system indicator, but I'm not completely hung up on it so `{root}` would work (on the basis that it looks like the templates lower down).

Reply:
As discussed on Slack, if I switch to `{root}` you need to quote the string in the YAML file. That seems more trouble than it's worth so I am sticking with the HTML-style tag.